src/app/header/header.component.ts

```diff
@@ -0,0 +1,35 @@
+import { CommonModule } from '@angular/common';
+import { Component, Input } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+
+@Component({
+  selector: 'app-header',
+  standalone: true,
+  imports: [CommonModule, FormsModule],
+  templateUrl: './header.component.html',
+  styleUrl: './header.component.css',
+})
+export class HeaderComponent {
+  @Input() subjectName: string = '';
+  @Input() subjectImage: string = '';
+  isDarkMode = false;
```

Comment:
add access modifier to this 'public'. `public isDarkMode = false;`

and Please remove all comments from the code